### ISSUE TITLE
Including .css files with @import is non-standard behaviour

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#6950]
Let's fix this warning:

> Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass.
> Use a custom importer to maintain this behaviour. Check your implementations documentation on how to create a custom importer.

It happens when running:

- `yarn storybook`
- `yarn build`

**Comment by HosseinMarzban** [comment]
hi @willdurand, According to [this issue](https://github.com/sass/node-sass/issues/2362) it's a "bug" from node-sass.
currently, we use node-sass  version 4.9.1, and base on [this comment](https://github.com/sass/node-sass/issues/2362#issuecomment-439012457) we have to wait for the new version which is 4.11.0.

**Comment by willdurand** [comment]
Hi @HosseinMarzban, yep I was about to comment too. The message was not a bug but it has been reverted because of the community's reaction: https://github.com/sass/libsass/pull/2684.

**Comment by HosseinMarzban** [comment]
Exactly, I read it all, so we have to wait until the new version comes up, then update the package.
